core/primitives/src/test_utils.rs

```diff
@@ -273,6 +277,52 @@ impl SignedTransaction {
         )
     }
 
+    pub fn deploy_global_contract(
+        nonce: Nonce,
+        contract_id: &AccountId,
```

Comment:
nit: internally you `clone()` account anyway, so it is better to take owned `AccountId` as an argument and let client call `.clone()` if needed, something Simonas taught me